composer.json

```diff
@@ -20,11 +20,22 @@
         "zicht/admin-bundle": "^5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4",
-        "zicht/standards-php": "*@stable"
+        "phpunit/phpunit": "^7",
+        "zicht/standards-php": "^3.3"
     },
     "autoload": {
         "psr-0": { "": ["src/", "tests/"] }
```

Comment:
Graag `tests` in `autoload-dev` section zetten, anders worden alle tests ook meegenomen in de classmap van een project bij autoloader optimization.
```
    "autoload": {
        "psr-0": { "": ["src/"] }
    },
    "autoload-dev": {
        "psr-0": { "": ["tests/"] }
    },
```